dotnet/src/SemanticKernel.Abstractions/AI/PromptExecutionSettings.cs

```diff
@@ -15,7 +16,7 @@ namespace Microsoft.SemanticKernel;
 /// if the service they are calling supports additional properties. For an example, please reference
 /// the Microsoft.SemanticKernel.Connectors.OpenAI.OpenAIPromptExecutionSettings implementation.
 /// </remarks>
-public class PromptExecutionSettings
+public class PromptExecutionSettings : ICloneable
```

Comment:
Implementing this interface is generally discouraged. You can still have a Clone method, but we generally don't implement the interface on new things anymore. One of those examples where the interface shipped without enough implementers and consumers to understand its real worth.